user/resetPassword/src/main/java/in/projecteka/resetpassword/ui/fragment/ResetPasswordOtpFragment.kt

```diff
@@ -107,6 +108,10 @@ class ResetPasswordOtpFragment : BaseFragment() {
 
         spannableString.setSpan(ForegroundColorSpan(Color.WHITE), 0, message.length, Spannable.SPAN_EXCLUSIVE_EXCLUSIVE)
         snackbar = Snackbar.make(snackbar_container, spannableString, Snackbar.LENGTH_LONG)
+
+        val layout = snackbar.view
+        val textView = layout.findViewById(com.google.android.material.R.id.snackbar_text) as TextView
```

Comment:
to prevent finding the same text view repeatedly, which is expensive , you can save the text view as an instance variable once it is found the first time. Also, do make sure you mark this as optional because the Id might change over a period of time.